terraform/modules/api/main.tf

```diff
@@ -81,6 +104,17 @@ resource "aws_api_gateway_rest_api" "scale" {
   }
 }
 
+# Default Access Denied gateway response exposes info about the API so replace it.
+resource "aws_api_gateway_gateway_response" "access_denied" {
+  rest_api_id   = aws_api_gateway_rest_api.scale.id
+  status_code   = "403"
+  response_type = "ACCESS_DENIED"
+
+  response_templates = {
+    "application/json" = jsonencode({ "message" = "Access denied" })
```

Comment:
Think it should be 'Your name's not on the list, you're not getting in', but happy with 'Access denied' 😄

Reply:
> Think it should be 'Your name's not on the list, you're not getting in', but happy with 'Access denied' 😄

😄 "Computer says No...."?